_episodes/03-create.md

```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
 > ~~~
 > {: .bash}
 >
-> Why is it a bad idea to do this?
+> Why is it a bad idea to do this? (Notice here that the `planets` project is now also tracking the entire `mars` repository.)
 > How can Dracula undo his last `git init`?
+> > ## Solution
+> > Git repositories can interfere with each other if they are "nested" in the
+> > directory of another. We therefore advise to create each new git repository in a
+> > separate directory. Note that we can track files in directories within a git
+> > repository as shown:
+> >
+> > ~~~
+> > touch moon phobos deimos titan    # create moon files
+> > cd ..                             # return to planets directory
+> > ls moons                          # list contents of the moons directory
+> > git add moons/*                   # add all contents of planets/moons
+> > git status                        # show moons files in staging area
+> > git commit -m "add moon files"    # commit planets/moons to planets git repo
+> > ~~~
+> > {: .bash}
+> >
+> > Similarly, we can ignore (as discussed later) entire directories, such as the moons directory:
+> >
+> > ~~~
+> > echo moons >> .gitignore           # ignore all files within moons
```

Comment:
Replace this with 

~~~
$ nano mars.txt
$ cat .gitignore
~~~

~~~
moons
~~~

This is only style.